### PR TITLE
bump to 3.5.17, update pins, clean blanks

### PR DESCRIPTION
**Pull request by sdvillal**
I wonder what could it take to build this for windows. Could we just use a [precompiled package](https://www.gnutls.org/download.html)?

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,14 +44,14 @@ requirements:
     - zlib 1.2.8
```
Comment (jakirkham) [inline recipe/meta.yaml]: These should be refreshed too.

**Comment by Cashalow** [comment]
Sorry I can't help you for the windows build !

**Comment by Cashalow** [comment]
Should I merge this yet or do you want to add the windows binary now?

**Comment by sdvillal** [comment]
Ready to merge, thanks for looking at this.

**Comment by jakirkham** [comment]
Thanks @sdvillal. Have you looked into using this with `ffmpeg` already?

**Comment by sdvillal** [comment]
@jakirkham Yeah, low hanging fruit. See https://github.com/conda-forge/ffmpeg-feedstock/pull/45

**Pull request merged**